pkgs/development/libraries/libgudev/default.nix

```diff
@@ -25,6 +24,18 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-YSZqsa/J1z28YKiyr3PpnS/f9H2ZVE0IV2Dk+mZ7XdE=";
   };
 
+  patches = [
+    # Conditionally disable one test that requires a locale implementation
+    # https://gitlab.gnome.org/GNOME/libgudev/-/merge_requests/31
+    ./tests-skip-double-test-on-stub-locale-impls.patch
+  ];
+
+  # The relative location of LD_PRELOAD works for Glibc but not for other loaders (e.g. pkgsMusl)
```

Comment:
Also please move the comment inside so it does not get disassociated when more commands are added.

Reply:
Done